packages/teams-js/src/public/runtime.ts

```diff
@@ -7,9 +7,25 @@ import { HostClientType } from './constants';
 
 const runtimeLogger = getLogger('runtime');
 
-export interface IRuntime {
+export interface IBaseRuntime {
   readonly apiVersion: number;
   readonly isLegacyTeams?: boolean;
+}
+
+/**
+ * Latest runtime interface version
+ */
+export type Runtime = IRuntimeV1;
```

Comment:
should this be renamed to LatestRuntime?

Reply:
It could, but most code using the `runtime` object shouldn't be concerned about versioning, so I think it might be more helpful to think of this as "this is the `Runtime` interface" rather than leaking the versioning terminology into the rest of the codebase. What do you think?

Reply:
okay, that's fine. It's internal, so easy to change if we change our minds